### PR TITLE
more selinux string fixes

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
...and a nice regexp helper.

### CODE REVIEW

File: test/verify/check-selinux [hunk 1]
```diff
@@ -192,7 +193,7 @@ class TestSelinux(MachineCase):
             # fallback for older Chrome releases
             b.grant_permissions("clipboardRead", "clipboardWrite")
 
-        b.wait_visible(".pf-c-data-list__cell:contains(Allow zebra to write config)")
+        b.wait_visible(".pf-c-data-list__cell:contains(Allow zebra)")
```
Comment (mvollmer) [inline test/verify/check-selinux:196]: Should this also be "Allow zebra.*write.*config" like above for .modifications-table?

Comment (allisonkarlitskaya) [reply test/verify/check-selinux:196]: Because we use the `:contains()` selector, we can't put a regexp here.

**Review by mvollmer — APPROVED** [review]

File: test/verify/check-selinux (hunk 1 above)
Comment (mvollmer) [reply test/verify/check-selinux:196]: Ouch, obviously.

**Comment by allisonkarlitskaya** [comment]
rhel-9-0 is clean against cockpit-project/bots#2306 so I'll land that one straight away.  fedora-33 on cockpit-project/bots#2313 has a (probable) flake so I'll rerun that batch, though.

**Pull request merged**